ui/packages/audio/src/Audio.svelte

```diff
@@ -57,7 +57,26 @@
 	}
 
 	async function prepare_audio() {
-		const stream = await navigator.mediaDevices.getUserMedia({ audio: true });
+		const stream = await navigator.mediaDevices
+			.getUserMedia({ audio: true })
+			.then((stream) => {
+				return stream;
+			})
+			.catch((err) => {
+				if (err instanceof DOMException && err.name == "NotAllowedError") {
+					alert(
+						"Please allow the use of microphone if you wish to record audio."
+					);
+					return null;
+				} else {
+					throw err;
+				}
+			});
```

Comment:
We shouldn't be mixing await and `then` chaining. Lets wrap this in a `try` `catch` block instead.

Additionally should think of something less intrusive than an `alert` which blocks all interaction with the page? cc @gary149 @abidlabs